scripts/mods/botania.zs

```diff
@@ -439,3 +439,72 @@ recipes.removeByRecipeName('botania:tornadorod');   recipes.addShaped(<botania:t
 recipes.removeByRecipeName('botania:waterrod');     recipes.addShaped(<botania:waterrod>, [[null, <minecraft:potion>.withTag({Potion: 'minecraft:water'})], [<ore:livingwoodTwig>, null]]);
 recipes.removeByRecipeName('botania:skydirtrod');   recipes.addShapeless(<botania:skydirtrod>, [<botania:dirtrod>, <ore:elvenPixieDust>]);
 // ////////////////////////////////////////////////////////////////
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "campanimia"}), [
+<botania:rune:1>,
+<botania:rune:2>,
+<botania:rune:3>,
+<botania:rune>,
+<quark:rune:14>,
+<quark:rune:2>,
+<quark:rune:4>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "nuclianthus"}), [
+<quark:rune:5>,
+<quark:rune:4>,
+<quark:rune:4>,
+<quark:rune:1>,
+<botania:manaresource:1>,
+<botania:rune:6>,
+<botania:rune:6>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "rokku_eryngium"}), [
+<botania:manaresource:5>,
+<botania:manaresource:9>,
+<botania:rune:11>,
+<botania:rune:12>,
+<quark:rune:11>,
+<quark:rune>,
+<quark:rune>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "jikanacea"}), [
+<astralsorcery:itemtunedcelestialcrystal>.withTag({astralsorcery: {constellationName: "astralsorcery.constellation.horologium", crystalProperties: {collectiveCapability: 100, size: 900, fract: 0, purity: 100}}}),
+<botania:rune:14>,
+<botania:rune:9>,
+<quark:rune:2>,
+<quark:rune:4>,
+<quark:rune:4>,
+<thaumicaugmentation:material:5>,
+]);
```

Comment:
I dont like this recipe.

1. Horologrium is too grindy and boring ingredient to make, especially considering it require lot of boring processes like waiting 28 days. Replace this ingredient to something common and widely used. It may be related to AS as well
2. I dont know, maybe its my personal feeling, but i have picking exact color for most of botania recipe, like picking petal colors. Its sounds repetative. I would ask here to replace exact colored runes to wildcarded ones, accepting any rune color.

![image](https://github.com/user-attachments/assets/48286d1d-a437-423b-af5e-cf5be3d132c4)

Reply:
1.Replaced crystal with resonating gem
2. Personally i think that crafting recipe or runes is awesome but never used in actual progress. We even have rainbow rune that is combination of all 8 craftable runes, I just feel it's kinda wasted crafting ingredient. I'm willing to replace all runes with petal to match classic botania petals